haystack/telemetry.py

```diff
@@ -338,6 +337,10 @@ def apply_filter(cls, param_dicts: Dict[str, Any]) -> Dict[str, Any]:
 
         # if model_name_or_path is a local file path, we reduce it to the model name
         if "model_name_or_path" in tracked_params:
-            if Path(tracked_params["model_name_or_path"]).is_file():
+            if (
+                Path(tracked_params["model_name_or_path"]).is_file()
+                or tracked_params["model_name_or_path"].count("/") > 1
```

Comment:
Sorry I just had a realization! On Windows this check won't work as expected (paths use \ rather than / ). Let's use `os.path.sep`?